ui/app/js/services/rules_service.js

```diff
@@ -53,6 +53,9 @@ angular.module("app").factory('Rules', function($http) {
       data.csrf_token = csrf_token;
       return $http.post("/api/scheduled_changes/rules", data);
     },
+    getScheduledByUser: function () {
+      return $http.get("/api/users/current");
```

Comment:
You should just use Permissions.getCurrentUser (https://github.com/mozilla/balrog/blob/3c6711fa6a7e72e641742ee610213a39533002d8/ui/app/js/services/permissions_service.js#L6) instead - there's no need to duplicate this here.